ecosystem/python/sdk/aptos_sdk/account.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Aptos
+# SPDX-License-Identifier: Apache-2.0
+
+from __future__ import annotations
+
+import json
+import tempfile
+import unittest
+
+import ed25519
+from account_address import AccountAddress
+
+
+class Account:
+    """Represents an account as well as the private, public key-pair for the Aptos blockchain."""
+
+    account_address: AccountAddress
+    private_key: ed25519.PrivateKey
```

Comment:
These should not be class variables. When you define the constructor:
    ```
def __init__(
        self, account_address: AccountAddress, private_key: ed25519.PrivateKey
    ):
        self.account_address = account_address
        self.private_key = private_key
```
The class variables are shadowed. You are actually initiating new instance variables that only exist in the object.

Reply:
https://stackoverflow.com/a/69115321

since these aren't specified with values, they just become place holders for defining instance types.